src/ruby/.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
In this PR, let's just instantiate the Basic instance in the test suite, and then run it on Travis. That's it. In further PRs, we will do more of course.